src/agentlessd/agentlessd.c

```diff
@@ -497,10 +501,13 @@ void Agentlessd()
 
             sleep(i);
         }
-
+#ifdef WAZUH_UNIT_TESTING
+        break;
+#else
         /* We only check every minute */
         test_it = 0;
         sleep(60);
+#endif
```

Comment:
Design nitpick: check `FOREVER()` implementation to avoid real infinite loops handling on UT

Reply:
ok I didn't saw, and it is a better implementation